src/utils/getTreasuryBalances.mjs

```diff
@@ -0,0 +1,128 @@
+import env from 'dotenv';
+import alasql from 'alasql';
+import fetch from "node-fetch";
+
+import {readFile, writeFile, mkdir} from 'fs/promises';
+
+let conf = env.config({path: '.env'});
+let etherscankey = conf.parsed.ETHERSCAN_API_KEY;
+
+// works with multiple treasuries
+// let treasuries = ["0x78605Df79524164911C144801f41e9811B7DB73D",
+//   "0x78605Df79524164911C144801f41e9811B7DB73D"];
+
+let treasuries = ["0x78605Df79524164911C144801f41e9811B7DB73D"];
+
+function getURL(action, address) {
+  let t = 'https://api.etherscan.io/api'
+      + '?module=account'
+      + '&action=' + action
+      + '&address=' + address
+      + '&startblock=0'
+      + '&endblock=99999999'
```

Comment:
Recommendation: set the startblock=12645922, the starting block of Bitdao contract